x-pack/plugins/apm/public/components/shared/license_prompt/index.tsx

```diff
@@ -54,9 +54,11 @@ export function LicensePrompt({ text, showBetaBadge = false }: Props) {
             'This feature is currently in beta. If you encounter any bugs or have feedback, please open an issue or visit our discussion forum.',
         }
       )}
+      description={<></>}
+      title={<></>}
```

Comment:
Can I make a suggestion for this file because this part looks a bit hacky since you're no longer using the card as intended. And truly it really is a card with different styles. But here's the gist of how I'd change it. The `display` prop will render correctly in the next EUI upgrade.

https://gist.github.com/cchaos/b7e7553126fe2e5de33e315d31c0d906